maintainers/maintainer-list.nix

```
@@ -16085,4 +16085,10 @@
     github = "ziguana";
     githubId = 45833444;
   };
+  snick = {
```

It's so much easier when this is the same as github nick ...

Also it seems the maintainers' list in this file is sorted, so please add at the correct position.

```suggestion
  realsnick = {
```

---

the maintainers list isn't sorted...just happen to look like that on few recent names